src/submit_jobs.py

```diff
@@ -123,17 +132,36 @@ def read_job(job: SparkJob):
         job.repartition()
 
     # Upload extracted files to AWS S3 in Hive-partitioned Parquet
+    aws = AWSClient(logger=session.logger)
+    new_local_files = []
```

Comment:
This part is a little wacky and I don't love it. Basically, we want to check if any of the jobs found new _local_ files during upload to S3. If they did, then we want to trigger a Glue crawler to pick up the new files/partitions. Open to a better design here though this does work.